features/kafka/producer/src/main/resources/OSGI-INF/blueprint/blueprint-kafka-producer.xml

```diff
@@ -17,7 +17,6 @@
       <cm:property name="eventTopic" value="events"/>
       <cm:property name="alarmTopic" value="alarms"/>
       <cm:property name="nodeTopic" value="nodes"/>
-      <cm:property name="topologyProtocols" value="bridge,cdp,isis,lldp,ospf"/>
```

Comment:
We also need to update the docs. See references to `topologyProtocols` in `opennms-doc/guide-admin/src/asciidoc/text/kafka-producer/kafka-producer.adoc`.